app/helpers/validate-send-helper.js

```diff
@@ -17,7 +17,7 @@ export default class ValidateSendHelper {
 
 	static isAccountBalanceId(v) {
 
-		const accountBalanceIdRegex = new RegExp(`^2\\.${constants.CHAIN_TYPES.IMPLEMENTATION_OBJECT_TYPE.ACCOUNT_BALANCE}\\.[1-9]+$`);
+		const accountBalanceIdRegex = new RegExp(`^2\\.${constants.CHAIN_TYPES.IMPLEMENTATION_OBJECT_TYPE_ID.ACCOUNT_BALANCE}\\.[1-9]+$`);
```

Comment:
You need to change regexp, because it doesn't work for 10 or 100, https://i.imgur.com/gM97Zdz.png